jax/experimental/jax2tf/jax2tf.py

```diff
@@ -636,7 +636,7 @@ def _lower_native_and_run(fun_jax: Callable,
   lowered = fun_jax_lower(*arg_specs_jax)._lowering
   mhlo_module = lowered.mhlo()
   mhlo_module_text = mlir.module_to_string(mhlo_module)
-  logging.vlog(2, f"XlaCallModule {mhlo_module_text}")
+  logger.warning(f"XlaCallModule {mhlo_module_text}")
```

Comment:
logger.debug

Also, string interpolation should be done via an argument, not an f-string. The point is you don't want to do the interpolation if the log isn't triggered.